packages/webapp/components/layouts/AccountLayout/Profile/index.tsx

```diff
@@ -370,7 +371,11 @@ const ProfileIndex = ({
     </form>
   );
 
-  if (isMobile) {
+  useEffect(() => {
```

Comment:
@capJavert knew we needed a hook 🪝 😂
![Screenshot 2024-10-23 at 14 19 55](https://github.com/user-attachments/assets/621b885c-0bd5-4c20-b823-5f3753d8cf06)

Reply:
Yep. I'm about to write one now 😂

Reply:
> No but we don't want it, its not really "hydrated" :hidethepain:, hydration actually happens before all of this and its just a hack, so if we add it I feel like its gonna be used in all sorts of things and that is bad.

Yeah, same thoughts, but I'm struggling to find a way to work this around. We might have to refactor the `MainLayout` component to strictly keep all components on render and just decide which ones to display based on size and some other parameters.

> We should strive to fix hydration issue by proper rendering on both side and only if not possible (like this sidebar issue we have currently) we can do the hack and in these cases its fine that code just repeats there, it should be intentional.

Agree with that. Let's see how we can fix this in a more proper way across the app 🚀 

> Also this hydrated pattern forces another re-render which is not ideal so we need to implement it in a way that it affects minimal component tree. Either way let's sync tomorrow 👍

Yes, this is basically what we tried to improve from the previous PRs 😂 😭